README.adoc

```diff
@@ -307,15 +307,15 @@ COPY --from=build ${DEPENDENCY}/BOOT-INF/classes /app
 ENTRYPOINT ["java","-cp","app:app/lib/*","hello.Application"]
 ----
 
-The first image is labelled "build" and it is used to run Maven and build the fat jar, then unpack it. The unpacking could also be done by Maven or Gradle (this is the approach taken in the Getting Started Guide) - there really isn't much difference, except that the build configuration would have to be edited and a plugin added.
+The first image is labelled "`build`" and it is used to run Maven and build the fat jar, then unpack it. The unpacking could also be done by Maven or Gradle (this is the approach taken in the Getting Started Guide) -- there really isn't much difference, except that the build configuration would have to be edited and a plugin added.
```

Comment:
"`build`" should be `build` (only backticks with no quotation marks) because it appears in a code file. The two hyphens to make an em dash should remain unchanged.